src/frontend/components/Icon/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { appData } from '../../data/appData';
+
+const Svg = styled.svg`
+  width: 1rem;
+  height: 1rem;
```

Comment:
What do you think to set `height` and `width` to `1rem` through `defaultProps` to be able to set it if needed?

Reply:
There's no use-case now so given it's trivial to implement when we need it, I'd rather wait to need it to do it.